cmd/bnscli/clitests/update_multisig_contract.test

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+bnscli multisig -update 000000000001 -activation 4 -admin 8 \
```

Comment:
This is 12 chars, 6 bytes... I thought the contract_id below looked funny.
Please pad to 16 chars (8 bytes)